src/index.js

```diff
@@ -1,47 +1,18 @@
 // import _ from 'lodash';
 import './style.css';
 
-const listArray = [
-  {
-    description: 'Workout at 11am',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Complete projects',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Revision of Work',
-    completed: true,
-    index: 3,
-  },
-];
+import { addList, displayList } from './modules/UI.js';
+import localGet from './modules/Store.js';
 
-listArray.forEach((listObj) => {
-  const { description, completed, index } = listObj;
-  const ulLists = document.querySelector('.ul-lists');
+const form = document.querySelector('.add-list');
+const todoTask = document.getElementById('input');
 
-  const list = document.createElement('li');
-  list.setAttribute('class', 'li-list');
-  list.setAttribute('id', index);
-  ulLists.appendChild(list);
+window.addEventListener('load', () => {
+  displayList();
+});
 
-  const checkCont = document.createElement('div');
-  checkCont.className = 'checkbox-cont';
-  list.appendChild(checkCont);
-
-  const checkbox = document.createElement('input');
-  checkbox.setAttribute('type', 'checkbox');
-  checkbox.checked = completed;
-  checkCont.appendChild(checkbox);
-
-  const paragraph = document.createElement('p');
-  paragraph.textContent = description;
-  checkCont.appendChild(paragraph);
-
-  const ellipsis = document.createElement('i');
-  ellipsis.setAttribute('class', 'fa fa-ellipsis-v');
-  list.appendChild(ellipsis);
+form.addEventListener('click', (event) => {
+  event.preventDefault();
+  addList(todoTask.value, false, localGet().length + 1);
+  displayList();
```

Comment:
- [ ] The functionality to add a to-do task is working, but unfortunately, there the `displayList()` function is not working properly because the tasks fail to show up on the screen after adding them. Please try to fix it 👍 
- [ ] I also noticed that each time I click in the input, an empty task gets saved to `localstorage`. Please look into this 👇 
![image](https://user-images.githubusercontent.com/46839548/221138013-d24bf7d7-20a6-4cf6-a65b-ff95308fa3e4.png)

**HINT:** This behaviour is happening because you have added a `click event` to the form. Therefore whenever the form is clicked, it saves an empty record in the `localstorage`. There are a couple of ways you can solve this issue: You could either use a submit event, or you could add the click event on the `.add-btn` of the form so that the record is only entered when the 'Add' button is clicked 😎 

- [ ] Also note that due to the fact that the tasks are not showing on the screen, it is impossible to test the edit and delete functionality. The next reviewer will test these features 👍